setup.py

```diff
@@ -22,57 +22,57 @@
         #   tensorboard needs >=41
         #   50.0 is broken: https://github.com/pypa/setupatools/issues/2353
         "setuptools>=41.0.0,!=50.0",
-        "cached-property",
-        "click",  # used in scl
+        "cached-property==1.5.2",
+        "click==8.0.3",  # used in scl
         "gym==0.18.3",
-        "numpy",
-        "pandas",
-        "psutil",
+        "numpy==1.19.5",
+        "pandas==1.3.4",
+        "psutil==5.8.0",
         "pybullet==3.0.6",
-        "pynput",  # Used by HumanKeyboardAgent
-        "rich",
-        "rtree",  # Used by sumolib
-        "sh",
-        "shapely",
-        "sklearn",  # KDTree from sklearn is used by sumo lanepoints
-        "tableprint",
+        "pynput==1.7.4",  # Used by HumanKeyboardAgent
```

Comment:
I believe particular dependency is very specific to the `human_in_the_loop.py` example. We should be able to take it out of the main dependencies.

Reply:
It is going to be removed for a different reason: https://github.com/huawei-noah/SMARTS/issues/1152